### PR TITLE
Add notes that master is a development branch in readme

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

**Comment by zmb3** [comment]
Yeah that looks like a good solution, thanks @stevenGravy

**Comment by stevenGravy** [comment]
updated @zmb3

**Review by zmb3 — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ The `teleport` repository contains the Teleport daemon binary (written in Go)
 and a web UI written in Javascript (a git submodule located in the `webassets/`
 directory).
 
+If your intention is to build and deploy for use in a production infrastructure 
+a released tag should be used.  The default branch, `master`, is the current 
+development branch for an upcoming major version.  Get the latest release tags 
+listed at https://goteleport.com/download/ and then use that tag in the `git clone`. 
+For example `git clone https://github.com/gravitational/teleport.gi -b v9.1.2` gets release v9.1.2.
```
Comment (r0mant) [inline README.md:133]:
```suggestion
For example `git clone https://github.com/gravitational/teleport.git -b v9.1.2` gets release v9.1.2. 
```

**Review by r0mant — APPROVED** [review]
A typo.

**Pull request merged**